vue-components/stories/Message.stories.ts

```diff
@@ -7,22 +7,7 @@ export default {
 	title: '/Message/Message',
 };
 
-export function all(): Component {
-	return {
-		components: { Message },
-		template: `
-			<div>
-				<p><Message type="error">An <b>error</b> which means it's potentially stopping whatever the user just did and providing them a clear message on what's wrong. There's also <a href="https://example.com">an explanatory link</a>.</Message></p>
-				<p><Message type="warning">A warning which is not stopping whatever the user just did, but needs to clear up why things might not work as expected.</Message></p>
-				<p><Message type="notice">A neutral message which inform about general higher priority notices to the user. Stay neutral!</Message></p>
-				<p><Message type="success">A success message which means that the user did everything right and should know about it. Yay!</Message></p>
-			</div>
-		`,
-	};
-}
-
-
-export function controllableMessage( args: { fullwidth: boolean, content: string, type: string } ): Component {
+export function message( args: { content: string; type: string } ): Component {
```

Comment:
I think we should follow the naming conventions @SaiSan-WMDE present to us in the [storybook structure document](https://docs.google.com/document/d/1loENZ13z5s_Nes4exm5Z9d5S3PnR7F_0brv5azQ5XB4/edit). In that case the story should be called `Basic`.

Reply:
Done.